substrate/frame/staking/src/lib.rs

```diff
@@ -1223,3 +1223,62 @@ impl BenchmarkingConfig for TestBenchmarkingConfig {
 	type MaxValidators = frame_support::traits::ConstU32<100>;
 	type MaxNominators = frame_support::traits::ConstU32<100>;
 }
+
+/// A result from a disabling decision made by [`DisablingStrategy`].
+pub struct DisablingDecision {
```

Comment:
Yes, in this case it just for future extension. At this point it is an overkill. But if we want to reenable offenders in the future `make_disabling_decision` will need to return a pair of vecs and it will be easy to mess up who is who. So I thought it's a better decision to introduce the struct now so it would be easier to extend it in the future.

But again I have no strong opinion on this. If you prefer to have a single `Vec` as a result - I'll do it.

Reply:
I would remove the `DisablingDecision` and replace it with `Vec<u32>` as it keeps the code simpler and it's more idiomatic based on the current code.